Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -1918,6 +1918,15 @@ theorem ofReal_le_ennnorm (r : ℝ) : ENNReal.ofReal r ≤ ‖r‖₊ := by
 
 end Real
 
+namespace NNReal
+
+instance : NNNorm ℝ≥0 where
```

Comment:
Does it make sense to add an instance for `NNRat` in this PR too?

Reply:
Do you have some applications in mind?